src/main/java/org/gridsuite/cgmes/boundary/server/fullmodel/CgmesNames.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2017-2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.cgmes.boundary.server.fullmodel;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
```

Comment:
Could you mention in the javadoc of these 3 copied classes that it has been copied from powsybl core and shouls be removed in 3.6.0?

Reply:
Done

Reply:
Done